taskcluster/ci/trigger-nightly/kind.yml

```diff
@@ -0,0 +1,47 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - ac_taskgraph.transforms.trigger_nightly:transforms
+    - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.task:transforms
+
+task-defaults:
+    run-on-tasks-for: [github-push]
+    run-on-git-branches: ['main$']
```

Comment:
Interesting. Does this mean it'll run on the branch `fake-main` as well?
Not a blocker.

Reply:
GMTA 🙂 I had a the same question in https://github.com/mozilla-mobile/android-components/pull/12793#discussion_r975295612. The answer is in the thread there.

Reply:
I think the 2 questions on this out of 2 reviews are clear evidence this is confusing, and so I'll change it to `^main$`.